client/my-sites/comments/index.js

```diff
@@ -7,26 +7,49 @@ import page from 'page';
 /**
  * Internal dependencies
  */
-import controller from 'my-sites/controller';
-import { clearCommentNotices, comments, redirect } from './controller';
+import { siteSelection, navigation, sites } from 'my-sites/controller';
+import { clearCommentNotices, comment, postComments, redirect, siteComments } from './controller';
 import config from 'config';
+import { VALID_STATUSES } from './constants';
 
 export default function() {
 	if ( ! config.isEnabled( 'comments/management' ) ) {
 		page( '/stats' );
 	}
 
 	if ( config.isEnabled( 'comments/management' ) ) {
-		page( '/comments/:status?', controller.siteSelection, redirect, controller.sites );
-
+		// Site View
 		page(
-			'/comments/:status/:site',
-			controller.siteSelection,
-			redirect,
-			controller.navigation,
-			comments
+			`/comments/:status(${ VALID_STATUSES.join( '|' ) })/:site`,
```

Comment:
Just a note that this will exclude `unapproved`. If we decided to keep them both maybe we could add it to `VALID_STATUSES` here? Anyway, I'm still leaning towards replacing all occurrences of `pending` with `unapproved`. ¯\\_(ツ)\_/¯

Reply:
@vindl I don't have any strong opinions regarding `pending` vs `unapproved`, but maybe it would be worth doing a search+replace (or search+append?) in a single separate PR.

@gwwar Done. Also fun fact: the explicit list is long exactly like the `join` expression.